src/main/scala/com/gu/zuora/crediter/CreditTransferService.scala

```diff
@@ -82,8 +84,6 @@ class CreditTransferService(command: CreateCreditBalanceAdjustmentCommand)(impli
         maybeId.foreach(id => logger.info(s"Successfully created Credit Balance Adjustment, ID: $id"))
         maybeId
       }).flatten
```

Comment:
for example, imagine if println returned a value that you had to force evaluation before it would actually cause the printing to happen